src/platform_impl/web/web_sys/gamepad_manager.rs

```diff
@@ -0,0 +1,109 @@
+use super::gamepad;
+use super::utils;
+use crate::event::device;
+use std::{cell::RefCell, rc::Rc};
+
+pub struct Shared(Rc<GamepadManager>);
+
+pub struct GamepadManager {
```

Comment:
Is there any way to take the GamepadManager that's in common between both backends and make it backend-agnostic?

Reply:
Done!